src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length--28 calendar--start-day--sun">
```

Comment:
In the checklist it is specified that we make the calendar  for 31 days. 
This is a small detail but it is specified for this task. 
```suggestion
    <div class="calendar calendar--month-length--31 calendar--start-day--sun">
```